src/test/java/cd/go/artifact/docker/registry/model/BuildFileArtifactPlanConfigTest.java

```diff
@@ -89,7 +90,13 @@ public void shouldErrorOutWhenFileDoesNotExist() throws UnresolvedPropertyExcept
         final ArtifactPlanConfig artifactPlanConfig = new BuildFileArtifactPlanConfig("random.json");
 
         thrown.expect(RuntimeException.class);
-        thrown.expectMessage(String.format("%s/random.json (No such file or directory)", agentWorkingDir.getAbsolutePath()));
+        if (SystemUtils.IS_OS_WINDOWS) {
+            thrown.expectMessage(String.format("%s\\random.json (The system cannot find the file specified)", agentWorkingDir.getAbsolutePath()));
```

Comment:
Use `File.pathSeparator` perhaps?

```
throw.expectMessage("%s%s/random.json....", agentWorkingDir.getAbsolutePath(), File.pathSeparator)
```

Reply:
You mean File.separator maybe, right? pathSeparator will be ';' on Windows, e.g.

Reply:
Yes `File.separator`. Sorry, I always get confused between them.

Reply:
Committed a new version :)